Nitrox.Launcher/Nitrox.Launcher.csproj

```diff
@@ -56,13 +56,40 @@
             <ReferenceOutputAssembly>true</ReferenceOutputAssembly>
             <ExcludeAssets>compile</ExcludeAssets>
         </ProjectReference>
+
         <ProjectReference Include="..\NitroxPatcher\NitroxPatcher.csproj">
-            <ReferenceOutputAssembly>true</ReferenceOutputAssembly>
+            <Name>NitroxPatcher472</Name>
+            <SetTargetFramework>TargetFramework=net472</SetTargetFramework>
+            <SkipGetTargetFrameworkProperties>true</SkipGetTargetFrameworkProperties>
+            <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
```

Comment:
ReferenceOutputAssembly should removed (default true) here otherwise AssetsTools.NET dll isn't copied to net472 folder (dunno why but :man_shrugging:)

Reply:
AssetsTools.NET isn't supposed to be inside net472, since it's a server dependency, not a In game one